R/todor.R

```diff
@@ -102,9 +105,37 @@ todor <- function(todo_types = NULL, search_path = getwd(), file = NULL) {
   processed <- lapply(files, function(x) process_file(x, todo_types))
   names(processed) <- files
   markers <- create_markers(processed)
-  build_rstudio_markers(markers)
+
+  if(!(output %in% c("markers","text"))){
+    stop("Output format not recognised. Available options are \"markers\" and \"text\"")
+  }
+
+  if (output == "text") {
+    return(build_report(markers))
+  } else {
```

Comment:
since we require a specific argument `marker`, this could be handled by ifelse and else could inform about available options is someone mistypes...

Reply:
Yep - I had tried to have some error handling in the `IF NOT IN THEN STOP` function in the lines just before this, but can definitely improve it!

Reply:
Have improved error handling, which will give the following error if the `output` arg is mis-specified:

```
Output format not recognised. Available options are "markers", "list" and "markdown"
```